test/macros.jl

```diff
@@ -203,10 +203,10 @@ facts("[macros] @constraints") do
         ref[i=1:3], y[1] + y[i] >= i
     end)
 
-    @fact conToStr(m.linconstr[1]) --> "x + y[1] $eq 1"
```

Comment:
There's *a lot* of these. Why don't we have a 1-argument form of ``con_str`` that uses ``REPLMode``?